src/main/java/sirius/db/mixing/properties/LocalDateProperty.java

```diff
@@ -141,4 +141,10 @@ public void describeProperty(JSONObject description) {
                        IndexMode::docValues,
                        description);
     }
+
+    @Override
+    public String getColumnDefaultValue() {
+        // not supported
```

Comment:
```suggestion
        // If a default value is present, this will commonly be a temporal value and thus not a constant
        // (and therefore suitable for a global default value). Therefore we ignore this here. Note that
        // @DefaultValue can still be used to provide a default value to the SQL schema generator).
```